lib/view/weeklyTaskView.dart

```diff
@@ -20,6 +20,7 @@ class WeeklyTaskViewState extends State<WeeklyTaskView> {
   DateTime today = getDateOnly(DateTime.now());
 
   @override
+
   /// Initializes the state of the widget
   void initState() {
     super.initState();
```

Comment:
I think no newline here would be better

STYLE GUIDE ANDREW

Reply:
It happened during a merge conflict fiasco lole. Any silliness is probably from that and/or the auto formatter

Reply:
we ran dart format